server/src/main/scala/tailcall/server/AdminGraphQL.scala

```diff
@@ -53,7 +53,7 @@ object AdminGraphQL {
   implicit val dynamicValueSchema: Schema[Any, DynamicValue] = new Schema[Any, DynamicValue] {
     override protected[this] def toType(isInput: Boolean, isSubscription: Boolean): __Type =
       __Type(kind = __TypeKind.SCALAR, name = Some("DynamicValue"))
-    override def resolve(value: DynamicValue): Step[Any] = Step.PureStep(DynamicValueUtil.toValue(value))
+    override def resolve(value: DynamicValue): Step[Any] = Step.PureStep(DynamicValueUtil.toResponseValue(value).get)
```

Comment:
We can return NullValue when toResponseValue is not possible.

Reply:
Done.